docs/fenix-custom-targeting.md

```diff
@@ -0,0 +1,33 @@
+---
+id: fenix-custom-targeting
+title: Adding new targeting parameters to Fenix
+---
+# Adding new targeting parameters to Fenix
```

Comment:
Nit, but can you use the word `attribute` v.s. parameter, since this is how it's referred to in a lot of existing documentation and in [desktop](https://firefox-source-docs.mozilla.org/browser/components/newtab/content-src/asrouter/docs/targeting-attributes.html)?